lib/recurly/client.php

```diff
@@ -62,6 +62,7 @@ class Recurly_Client
   const PATH_COUPON_REDEMPTION = '/redemption';
   const PATH_COUPON_REDEMPTIONS = '/redemptions';
   const PATH_COUPONS = '/coupons';
+  const PATH_GIFT_CARDS = '/gift_cards';
```

Comment:
What do you think the original intention was? Keeping them all in one place? Could they be referenced from different classes? It seems to me like it's better for the endpoint to live with the class responsible for maintaining it. It would be nice to understand why this was done though. I'm cool with changing it if you can't think of a reason why it should stay like this.